.gitignore

```diff
@@ -1,5 +1,6 @@
 bin/
 bin64/
+build/
```

Comment:
This has to go in your personal .gitignore, because "build/" is a well-defined Boost directory. For example: https://github.com/boostorg/json/tree/develop/build

This commit needs to be removed from the pull request.

Reply:
Done